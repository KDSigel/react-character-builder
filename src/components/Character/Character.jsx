import React from 'react'

export default function Character(head) {
    return (
        <>
        <div style={{
            backgroundImage: `url(${head}.png)`,
            height: 250,
            width: 788,
            backgroundSize: 'cover',
            backgroundRepeat: 'no-repeat',
        }}
        >
        </div>
        <div style={{
            backgroundImage: 'url(torso-1.png)',
            height: 400,
            width: 788,
            backgroundSize: 'cover',
            backgroundRepeat: 'no-repeat',
        }}
        >
        </div>
        <div style={{
            backgroundImage: 'url(legs-1.png)',
            height: 785,
            width: 788,
            backgroundSize: 'cover',
            backgroundRepeat: 'no-repeat',
        }}
        >
        </div>
        </>
    )
}
