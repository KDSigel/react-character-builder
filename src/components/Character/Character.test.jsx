import {screen, render} from '@testing-library/react';
import Character from './Character';

it('should display the Character component', () => {

  const {container} = render(<Character />);
  expect(container).toMatchSnapshot();
  
});