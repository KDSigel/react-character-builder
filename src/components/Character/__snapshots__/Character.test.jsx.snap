// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the Character component 1`] = `
<div>
  <div
    style="background-image: url(undefined.png); height: 125px; width: 394px; background-size: cover; background-repeat: no-repeat;"
  />
  <div
    style="background-image: url(undefined.png); height: 200px; width: 394px; background-size: cover; background-repeat: no-repeat;"
  />
  <div
    style="background-image: url(undefined.png); height: 392.5px; width: 394px; background-size: cover; background-repeat: no-repeat;"
  />
</div>
`;
